crypto/crypto.go

```diff
@@ -1,21 +1,20 @@
 package crypto
 
 import (
+	"crypto"
 	"crypto/rand"
 
 	"golang.org/x/crypto/ed25519"
 	"golang.org/x/crypto/sha3"
 )
 
 const (
-	HashSizeByte = 32
-	HashID       = "SHAKE128"
+	HashSizeByte   = 32
+	HashID         = "SHAKE128"
+	PrivateKeySize = 64
```

Comment:
It would be used outside the library. An example is when the developer reads the signing key from a file and he wants to validate the key size before using it.